dls-controls/malcolmjs
# Bugfix/ws reconnect #280

## Description

Fixed websocket reconnecting behaviour

## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- "npm run ui-only" and "npm run server" in 2 separate terminals
- navigate to localhost:3000/PANDA:INENC1/val
- let the plot gather a few seconds of data
- kill the window running the "npm run server" command
- check all the attributes (and plot) switch to the disconnected alarm state
- re-run the server
- after a few seconds, the gui should reconnect
- check that all the attributes come out of the disconnected state

## Agile board tracking

connect to #280 
closes #280
